### ISSUE TITLE
Doc about replacement cycling attacks

### ISSUE THREAD

**Issue by hieblmi** [t-bast/lightning-docs#28]
Hi @t-bast,

I am sure that your pages about lightning are a great learning resource for many lightning devs and enthusiats.
It would be great to see a document in this repo about mempool replacement cycling attacks that everyone got aware of now through [this post](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2023-October/021999.html), maybe in a similar fashion as you describe pinning attacks.

All the best.

**Comment by t-bast** [comment]
Good idea! A detailed post about mempool subtleties and their impact on L2 protocols would be generally useful, I'll do that when I find some time.
